src/transformState.js

```diff
@@ -5,7 +5,27 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  actions.forEach((el) => {
```

Comment:
p3. Try to use proper variable names in `for in` loops or in `forEach`
```suggestion
  actions.forEach((action) => {
```